doc/nrf/config_and_build/dfu/dfu_downgrade_protection.rst

```diff
@@ -0,0 +1,106 @@
+.. _ug_fw_update_image_versions_mcuboot_downgrade:
+.. _ug_fw_update_downgrade_protection:
+
+Downgrade protection
```

Comment:
Why is this part of DFU and not bootloader configuration?

Reply:
It was part of the fw_update.rst, hence it ended up in this section.